R/Spectra.R

```diff
@@ -971,11 +1001,16 @@ NULL
 #'     to import spectrum data from the provided files. See section *Creation
 #'     of objects, conversion and changing the backend* for more details.
 #'
-#' @param spectraVariables For `selectSpectraVariables`: `character` with the
+#' @param spectraVariables
+#' - For `selectSpectraVariables`: `character` with the
 #'     names of the spectra variables to which the backend should be subsetted.
-#'     For `addProcessing`: `character` with additional spectra variables that
+#' - For `addProcessing`: `character` with additional spectra variables that
 #'     should be passed along to the function defined with `FUN`. See function
 #'     description for details.
+#' - For `filterRanges` and `filterValues`: A `character` vector specifying the
```

Comment:
maybe also mention:

... vector specifying the column(s) from `spectraData(object)` on which to filter the data and that correspond to the ...